### DISCUSSION TITLE
Do we need to close a Discussion after it has been answered

### DISCUSSION THREAD

**Discussion by brsanthu** [eclipse-langium/langium#1484] [Q&A] [answered]
I see some folks closing the discussions after it has been answerd. By default Github doesn't show the Discussion if it is closed. 

What is the guideline?

**Comment by msujew** [comment, marked as answer]
I'm personally not a big fan of closing a discussion, as it makes it hard to search for a discussion IMO. I'm not even sure why there is the possibility of closing a discussion, since it's not actionable (unlike an issue, where it makes sense to close it).

If people close it themselves, that's fine. But we won't close discussions on our own.

**Reply by brsanthu** [reply]
Thank you, that's a reasonable guideline to follow that don't close the discussion unless discussion is really not useful somebody else.
